woocommerce/woocommerce-ios
# Simplified login: Integrate the mimimal login screens

Part of #7899, Closes: #7900 
Also closes #7901.

Please also review https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/691.


### Description

This PR updates the prologue, email, and password screen for the simplified login flow. Most of the changes are minor so I handled them all together to avoid having to release multiple versions of the `WordPressAuthenticator` library.

Figma design can be found here: ERGtCFLyiC9CFpHsw1XIid-fi-392%3A3599

Changes:
- A temporary feature flag for simplified login flow iteration 1 was added. This should be replaced with A/B testing after the updates are completed.
- New copies for the flow: the titles for the button on the prologue screen and instructions for the email screen.
- Update configs for `WordPressAuthenticator` to enable updates for the simplified login flow.
- New placeholder for the email field
- An empty implementation of the site creation entry point from the secondary button on the prologue screen.

### Testing instructions

- Build the app and log out if needed.
- Notice that the prologue screen shows only the Log In button (the site creation button is disabled for now)
- Tap the Log In button, and notice that the email screen is shown with the design and copies matching Figma. The "What is WordPress.com" link should not be visible either.
- Enter your email and tap Continue. Notice that the password screen is presented - the UI should match the Figma design as well minus the gray background beneath the avatar and email address. I'm holding that off because it takes more time, and the designers haven't settled on the idea.

With the feature flag `simplifiedLoginFlowI1` off, the flow should be exactly as before.

### Screenshots


https://user-images.githubusercontent.com/5533851/197748176-21ddea35-4ced-4b26-b8ba-0d116f67778c.mp4



---
